src/graphene-matrix.c

```diff
@@ -746,7 +746,25 @@ graphene_matrix_get_value (const graphene_matrix_t *m,
       return 0.f;
     }
 
-  return graphene_simd4f_get (r, col);
+  switch (col)
```

Comment:
Since this has nothing to do with ARM NEON issues, could you split this chunk into its own separate PR?